.travis.yml

```diff
@@ -1,10 +1,19 @@
 language: csharp
-mono:
-  - 4.8.1
+sudo: required
+dist: trusty
+dotnet: 1.0.0-preview2-003121
```

Comment:
This is to use this version: https://github.com/dotnet/netcorecli-fsc/wiki/.NET-Core-SDK-preview2 of dotnet cli

I used it because I think it is the one that introduces csproj back into the mix. Its not the latest preview (which is 2.0 preview 1). I believe this one matches what comes with VS 2017 out of box.